docs/sources/k6/next/testing-guides/calculate-concurrent-users.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Calculate concurrent users for load tests'
+description: 'Learn how to use Google Analytics data to determine the optimal number of concurrent users for your load tests.'
+weight: 200
+---
+
+# Calculate concurrent users for load tests
+
+When planning load tests, determining the right number of concurrent users is crucial for creating realistic test scenarios. This guide explains how to use Google Analytics data to calculate concurrent user numbers and design effective load tests.
+
+{{< admonition type="note" >}}
+
+You can use any analytics tool that provides the same metrics to help you calculate the concurrent users you need for load testing.
+
+{{< /admonition >}}
+
+## Understanding concurrent users
```

Comment:
Should we explain _what is_ a concurrent user? An easy-path might be to just link Wikipedia https://en.wikipedia.org/wiki/Concurrent_user